{{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py

```diff
@@ -181,6 +185,56 @@ class ProjectDefault(Configuration):
 
     SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
 
+    # Django REST Framework
+    # https://www.django-rest-framework.org/api-guide/settings/
+
+    REST_FRAMEWORK: dict = {
+        "DEFAULT_AUTHENTICATION_CLASSES": [
+            "rest_framework.authentication.SessionAuthentication",
+        ],
+        "DEFAULT_FILTER_BACKENDS": [
+            "django_filters.rest_framework.DjangoFilterBackend"
+        ],
+        "DEFAULT_PARSER_CLASSES": [
+            "djangorestframework_camel_case.parser.CamelCaseMultiPartParser",
+            "djangorestframework_camel_case.parser.CamelCaseJSONParser",
+        ],
+        "DEFAULT_PERMISSION_CLASSES": ["rest_framework.permissions.IsAuthenticated"],
+        "DEFAULT_RENDERER_CLASSES": [
+            "djangorestframework_camel_case.render.CamelCaseJSONRenderer",
+        ],
+        "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
+    }
```

Comment:
Add also this settings
`"JSON_UNDERSCOREIZE": {"no_underscore_before_number": True},`

By default djangorestframework-camel-case adds an underscore before the numbers and this often leads to unwanted effects like in this issue:
https://github.com/vbabiy/djangorestframework-camel-case/issues/50#issuecomment-462238378